rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -529,6 +529,15 @@ QString emu_settings::GetLocalizedSetting(const QString& original, emu_settings_
 		case video_renderer::vulkan: return tr("Vulkan", "Video renderer");
 		}
 		break;
+	case emu_settings_type::ShaderMode:
+		switch (static_cast<shader_mode>(index))
+		{
+		case shader_mode::shader_compiler: return tr("Shader Compiler", "Shader Mode");
```

Comment:
Maybe we shouldn't name this "Shader compiler" afterall, it doesn't really convey how different it is from the async compiler and how awful it actually is to use.
Maybe "Legacy shader compiler" would be more fitting? I think 'single threaded compiler' or 'inline shader compiler' would be more fitting, but even that does not convey how awful it actually is to use this mode.